usr/src/cmd/mdb/intel/kmdb/kaif.c

```diff
@@ -26,9 +26,9 @@
  */
 
 /*
- * The debugger/"PROM" interface layer
+ * The debugger/"PROM debugger" interface layer
```

Comment:
Also upstream first?

Reply:
I was hoping to do this as part of trying to clear up these layering descriptions, but ran into a wall because they don't make sense, and I've been having to figure out what they were probably meant to be based on how they acted on SPARC.

I left the minor cleanups behind on the basis that they improve things, and don't distract.

I can split from this commit, but I'm not sure I can upstream preductively.